### PR TITLE
Add Italian translation for kubectl

### PR DESCRIPTION
**Pull request by lucab85**
**What this PR does / why we need it:**
This PR provides first attempt to translate kubectl in Italian (related to #40645, #45573,
#45562, #40591, #46559).

**Which issue this PR fixes** _(`optional, in fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)_: fixes #
No issues

**Special notes for your reviewer:**
This PR requires Italian people to assist in the review. I'm native in Italian with BS in Computer Science.

```release-note
Adding Italian translation for kubectl
```

### CODE REVIEW

File: pkg/util/i18n/i18n.go [hunk 1]
```diff
@@ -38,6 +38,7 @@ var knownTranslations = map[string][]string{
 		"zh_CN",
 		"ja_JP",
 		"zh_TW",
+                "it_IT",
```
Comment (cblecker) [inline pkg/util/i18n/i18n.go]: There's an extra whitespace here that isn't needed.

**Review by cblecker — COMMENTED** [review]
@lucab85 This is worth adding a release note. Could you please add a release note to the body of your PR (something like the below, but without the leading + signs).
```
+```release-note
+Adding it_IT translation for kubectl
+```
```

**Comment by lucab85** [comment]
Please help me to solve the problem. I've tried to do `hack/generate-bindata.sh` as suggested in [translations/README.md](https://github.com/kubernetes/kubernetes/blob/master/translations/README.md) but suck with this error:

```
$ ./hack/generate-bindata.sh 
~/prj/github/kubernetes ~/prj/github/kubernetes
flag provided but not defined: -nometadata
Usage: go-bindata [options] <input directories>

  -debug=false: Do not embed the assets, but provide the embedding API. Contents will still be loaded from disk.
  -nocompress=false: Assets will *not* be GZIP compressed when this flag is specified.
  -nomemcopy=false: Use a .rodata hack to get rid of unnecessary memcopies. Refer to the documentation to see what implications this carries.
  -o="./bindata.go": Optional name of the output file to be generated.
  -pkg="main": Package name to use in the generated code.
  -prefix="": Optional path prefix to strip off asset names.
  -tags="": Optional set of uild tags to include.
  -version=false: Displays version information.
```
I'm using Ubuntu 16.04 + `longsleep/golang-backports` :
```
$ go version
warning: GOPATH set to GOROOT (/usr/lib/go-1.8/) has no effect
go version go1.8.3 linux/amd64
$ go-bindata --version
go-bindata 3.1.0 (Go runtime go1.2.1).
Copyright (c) 2010-2013, Jim Teeuwen.
```

**Comment by cblecker** [comment]
Try a `make generated_files`

**Comment by lucab85** [comment]
Thank you @cblecker for your hint but doesn't solve my issue.
The script `hack/generate-bindata.sh` stuck on lines 46 and 69 invoking `go-bindata` with `-nometadata` option witch doesn't exist in the latest version of  (3.0.7) so I removed this option.
The I was able to run the script and generate the `pkg/generated/bindata.go` file.

**Comment by brendandburns** [comment]
@lucab85 thanks for the PR. I'll look into either using an older version of go-bindata, or removing that flag...

**Comment by brendandburns** [comment]
@lucab85 

Actually, I think if you re-run `./hack/generate-bindata.sh` now that you patched the script, and commit the results it _should_ work...

Let us know if it doesn't.

**Comment by lucab85** [comment]
Thank you for your messages @brendandburns. 
My latest commit (`3c5f15f`) already included the result of `./hack/generate-bindata.sh` but unfortunately the build still **failed** as you can see above.
I hope somebody help me to address the issue.

**Comment by cblecker** [comment]
I think the version is off though.. it needs to match whatever CI is using so that it doesn't generate a diff

**Comment by lucab85** [comment]
Please help me to understand the issue in the latest [CI report](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/50155/pull-kubernetes-verify/45898/).
Thank you in advance.

**Comment by cblecker** [comment]
I think it's because you're using a version of `go-bindata` compiled with go1.2.1.

Try running `go get -u github.com/jteeuwen/go-bindata/...` to download/compile/install, and then run a `go-bindata -version` to verify that your version matches:
```
$ go version && go-bindata -version
go version go1.8.3 darwin/amd64
go-bindata 3.1.0 (Go runtime go1.8.3).
Copyright (c) 2010-2013, Jim Teeuwen.
```
Lastly, 3.1.0 does support the `-nometadata` flag, so your changes to `hack/generate-bindata.sh` can be reverted.

**Comment by cblecker** [comment]
Looks like CI is passing now. Please squash your commits :)
Thanks for sticking with this!

**Comment by brendandburns** [comment]
Yeah, thanks for doing this, if you squash commits (`git rebase -i HEAD~6`) I'm happy to lgtm and merge.

**Comment by lucab85** [comment]
Thank you @cblecker and @brendandburns for your support.
I hope that my squashed rebase with commit-id `96de8b9` works fine.

**Comment by cblecker** [comment]
@lucab85 It doesn't look like it worked, as there are 8 commits if you look here: https://github.com/kubernetes/kubernetes/pull/50155/commits

Another way to do it:
Looking at your branch (https://github.com/lucab85/kubernetes/commits/i18n-Italian), the last commit from the upstream is https://github.com/lucab85/kubernetes/commit/9aa04c755fc8091ce5a59d0087fe6a4bc0119195

So the following should work:
```
# rebase off of that last upstream commit
git rebase -i 9aa04c7
# The commit named "Italian translation" (https://github.com/lucab85/kubernetes/commit/96de8b93adfc38dbdb31a06963c883d2e95435bb) should be set to "pick"
# All the "Merge Branch" commits can be set to "drop"
```

Try that and see if it works for you.

**Comment by lucab85** [comment]
Thank you @cblecker. Wait  for CI tests for my squashed commit `1044b0d`.

**Comment by lucab85** [comment]
Yay @cblecker!

**Comment by cblecker** [comment]
/lgtm
over to @brendandburns for approval

**Comment by lucab85** [comment]
Thank you again for your support and help @cblecker and @brendandburns

**Comment by cblecker** [comment]
You're welcome, and thanks for contributing :)

**Comment by cblecker** [comment]
commenting to kick gubernator

**Pull request merged**